telegram/_inline/inlinekeyboardbutton.py

```diff
@@ -145,6 +145,12 @@ class InlineKeyboardButton(TelegramObject):
             be launched when the user presses the button.
         pay (:obj:`bool`): Optional. Specify :obj:`True`, to send a Pay button.
 
+    .. seealso:: `inlinekeyboardbutton.py <https://github.com/python-telegram-bot/
+        python-telegram-bot/blob/master/telegram/_inline/inlinekeyboardbutton.py>`_,
```

Comment:
```suggestion
    .. seealso:: 
```
can you move this above versionchanged directive? Apply to others too.
Also no need of linking to source code here. Only the examples is fine.